src/services/user-group/logic/courses.js

```diff
@@ -0,0 +1,8 @@
+
+const getAllCourseUserIds = ({ userIds = [], teacherIds = [], substitutionIds = [] }) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected block statement surrounding arrow body; move the returned value immediately after the `=>`.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5369432)